bin/pakiti-client

```diff
@@ -688,6 +725,10 @@ send the prepared report to the collecting server at the given URL
 
 set the path of the C<wget> command to use
 
+=item B<--svmon>, B<-sv>
```

Comment:
- The short-option (prefixed with a single -) must be a single letter
- Please retain the alphabetic order of the CLI parameters